course/layouts/shortcodes/choice.html

```diff
@@ -0,0 +1,5 @@
+{{ $correct := .Get "isCorrect" }}
```

Comment:
I have some concerns about how generic the name of this and the "choices" shortcode are.  If we were to implement a more generic "multiple choice" type shortcode then we might want to use these names.  Since these refer specifically to the quizzes, could we name them something like `quiz_choice` and `quiz_choices`?  Similarly, I think `multiple-choice-question` could be named `quiz_question`.  I think we should be using underscore separators in the filenames.  I realize that video gallery is `video-gallery.html` but it should be `video_gallery.html` and I need to make an issue to change that.  It can be kind of tricky to rename these after the fact because so much content has to be republished, so it's best to really have good names before committing anything.  Also note that there can be partials and shortcodes with the same name, since shortcodes are only called from within markdown content and partials are only called inside of layouts.